src/features/device-state/index.ts

```diff
@@ -45,6 +46,12 @@ export const setup = (app: Application) => {
 		statePatchV2,
 	);
 	app.patch('/device/v3/state', apiKeyMiddleware, statePatchV3);
+	app.get('/device/v3/fleet-state/:fleetUuid', apiKeyMiddleware, fleetStateV3);
```

Comment:
I know this was discussed, so is just a nitpick, but how about

`/device/v3/state/fleet/:fleetUuid`

or

``/device/v3/state/from-fleet/:fleetUuid`

Using `fleet-state` is not totally correct, since in the end the endpoint returns the device state as inherited from the fleet state.

Reply:
I'm also in favour of `/device/v3/state/fleet/:fleetUuid`

Reply:
cf this conversation : https://github.com/balena-io/open-balena-api/pull/1081#discussion_r1011501666